tests/core/test_context.py

```diff
@@ -329,3 +329,15 @@ def test_nested_str_default(
     assert parse_obj_as(model, {}).m == [sim_motor]  # type: ignore
     empty_context.device(alt_motor)
     assert parse_obj_as(model, {"m": [ALT_MOTOR_NAME]}).m == [alt_motor]  # type: ignore
+
+
+def test_plan_models_not_auto_camelcased(empty_context: BlueskyContext) -> None:
```

Comment:
This test doesn't really test anything to do with camelCasing. It passes without this change as the casing stuff creates aliases.

Reply:
Good spot, have amended the test and checked it fails without the change.